app/services/idv/steps/upload_step.rb

```diff
@@ -83,7 +83,14 @@ def mobile_device?
       end
 
       def form_response(destination:)
-        FormResponse.new(success: true, errors: {}, extra: { destination: destination })
+        FormResponse.new(
+          success: true,
+          errors: {},
```

Comment:
Not strictly necessary or relevant, but since we're here: `error` is [optional](https://github.com/18F/identity-idp/blob/6f2e33d9533cdaac8f1c8cce801ba5bd40649601/app/services/form_response.rb#L6), could we just drop it?

```suggestion
```